docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -140,20 +140,62 @@ Regardless of the way you submit your work, follow these steps.
 .. _ticket tracker: https://code.djangoproject.com/
 .. _Development dashboard: https://dashboard.djangoproject.com/
 
-Non-trivial contributions
-=========================
+Contributions which require community feedback
+==============================================
 
-A "non-trivial" contribution is one that is more than a small bug fix. It's a
-change that introduces new Django functionality and makes some sort of design
-decision.
+A patch that introduces Django functionality and makes some sort of design
+decision requires a wider community discussion. Especially if the approach
+involves a deprecation or potentially introducing breaking changes.
+Please include evidence that the approach and alternatives have been discussed
+on the `Django Forum`_ or |django-developers| list.
 
-If you provide a non-trivial change, include evidence that alternatives have
-been discussed on the `Django Forum`_ or |django-developers| list.
+Here are a few approaches for gaining feedback from the community.
 
-If you're not sure whether your contribution should be considered non-trivial,
-ask on the ticket for opinions.
+Django Forum or the django-developers mailing list
+--------------------------------------------------
+
+You can make a proposal on the `Django Forum`_ or |django-developers| mailing list.
+
+You should explain the need for the change, go into details of the approach
+and discuss alternatives. Please include a link to such discussions in your patch.
```

Comment:
(thinking aloud) there is an FAQ section which has a "how/when can I remind people about a change": https://docs.djangoproject.com/en/5.1/faq/contributing/#when-and-how-might-i-remind-the-team-of-a-change-i-care-about
This isn't an exact match but some people need to know it might take time to get feedback :thinking:

Reply:
> You should explain the need for the change, go into details of the approach
> and discuss alternatives. Please include a link to such discussions in your patch.
> Keep in mind that it might take some time to receive feedback, but you can remind people if it takes too long.[Add the link to the forum here]

Does this work?